en/docs/services/create-your-first-service.md

```diff
@@ -1,168 +1,161 @@
 # Create Your First Service
 
-To learn how to create a service, let's try a simple scenario. In this example, you are creating a service that consumes a public API that fetches the current active COVID-19 cases in a specified country, testing it, deploying it, and then observing its performance.
+To learn how to create a service, let's look at a simple scenario. In this example, you'll create a service that consumes a public API that fetches the current active COVID-19 cases in the United States. Then you'll test it, deploy it, and observe its performance.
```

Comment:
Shouldn't we add a similar description for the create and publish an API page?

Reply:
In the `Create and Publish Your First API` page we have a prerequisites section that describes the mock rest service used in the quick start. If we are to have a similar section we might have to consider removing that and having a similar description. Else there can be unnecessary repetition. WDYT?